src/containers/SearchPage.jsx

```diff
@@ -82,14 +82,15 @@ export class SearchPage extends React.Component {
               onSubmit={this.handleSubmit}
             />
             <hr />
-            <SearchResultsBox
-              show={submitted}
-              searchTerm={submittedSearchTerm}
-              fetchingInProgress={fetching}
-              sortedIds={sortedIds}
-              results={data}
-              getAdditionalInfo={getAdditionalInfo}
-            />
+            { submitted ? (
```

Comment:
Check out the ternary after this block on line 96. It looks like it does the same conditional check but renders something on the opposite case. I think we should merge these to just be:

```
{ submitted ? (
  <SearchResultsBox
    searchTerm={submittedSearchTerm}
    fetchingInProgress={fetching}
    sortedIds={sortedIds}
    results={data}
    getAdditionalInfo={getAdditionalInfo}
  />
) : <AppInfo /> }
```